cmd/images.go

```diff
@@ -0,0 +1,225 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"time"
+	"unicode"
+
+	"github.com/containers/common/pkg/report"
+	"github.com/containers/podman-bootc/pkg/utils"
+	"github.com/containers/podman/v5/pkg/bindings/images"
+	"github.com/containers/podman/v5/pkg/domain/entities"
+	"github.com/distribution/reference"
+	"github.com/docker/go-units"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	imagesCmd = &cobra.Command{
+		Use:   "images",
+		Short: "List bootc images in the local containers store",
+		Long:  "List bootc images in the local container store",
+		RunE:  doImages,
+	}
+)
+
+func init() {
+	RootCmd.AddCommand(imagesCmd)
+}
+
+func doImages(flags *cobra.Command, args []string) error {
+	machine, err := utils.GetMachineContext()
+	if err != nil {
+		println(utils.PodmanMachineErrorMessage)
+		logrus.Errorf("failed to connect to podman machine. Is podman machine running?\n%s", err)
```

Comment:
I'm not sure if we do this elsewhere in the code but IMO logging *and* returning errors often leads to unnecessary double printing.

Reply:
I remove these `println` when I see them